src/components/manage-registry/manage-agents/ManageAgents.tsx

```diff
@@ -58,7 +75,7 @@ export function ManageAgents({
     mutateAsync: mutateIdentityRegistryAddAgent,
     isPending: isAddAgentPending,
   } = useMutation({
-    mutationFn: async () => {
+    mutationFn: async (newUserAgentAddress: string) => {
```

Comment:
Why pass the arg and not use the one we have in state as it used to be?

Reply:
> Why pass the arg and not use the one we have in state as it used to be?

Yeah, because for now we can decide from where to get this new agent value: there are to different actions now.